PREP_SYSTEM_FOR_DIETPI.sh

```diff
@@ -507,8 +498,8 @@ _EOF_
 	if (( $G_HW_ARCH == 10 )); then
 
 		aPACKAGES_REQUIRED_INSTALL+=('linux-image-amd64')
-		aPACKAGES_REQUIRED_INSTALL+=('intel-microcode')
-		aPACKAGES_REQUIRED_INSTALL+=('amd64-microcode')
+		grep 'vendor_id' /proc/cpuinfo | grep -qi 'intel' && aPACKAGES_REQUIRED_INSTALL+=('intel-microcode')
```

Comment:
Not sure this would work on my Z83 (https://www.gearbest.com/tv-box-mini-pc/pp_562950.html)
```
root@DietPi:~# grep 'vendor_id' /proc/cpuinfo | grep -qi 'intel'; echo $?
0
root@DietPi:~# grep 'vendor_id' /proc/cpuinfo
vendor_id       : GenuineIntel
vendor_id       : GenuineIntel
vendor_id       : GenuineIntel
vendor_id       : GenuineIntel
```

I think there is too much variance in ```/proc/cpuinfo```, based on kernel, for us to successfully guarantee a valid detection, on all x86_64 devices?

Reply:
@MichaIng 
Ignore the above 😄 
```
-i, --ignore-case 
```